app/Http/Resources/UserBaseResource.php

```diff
@@ -33,7 +33,6 @@ public function toArray($request): array {
             'likes_enabled'  => $this->likes_enabled,
             $this->mergeWhen(isset($this->UserSettingsResource),
                 [
-                    'role'                    => $this->role,
```

Comment:
You can't delete this one yet. It will break some third-party-apps. Please set the return value to 0 and mark it as deprecated in the api-docs/inform our api consumers.